include/boost/url/impl/ipv4_address.ipp

```diff
@@ -116,6 +116,9 @@ print_impl(
                 *dest++ = '0' +
                     v / 100;
                 v %= 100;
+                *dest++ = '0' +
+                    v / 10;
+                v %= 10;
             }
             if(v >= 10)
```

Comment:
shouldn't this be "else" ?

Reply:
I'm not sure I understand where the `else` would be. If the number is greater than or equal to 100, it has 3 digits on the string no matter what.

Reply:
```
if( v >= 100 )
{
  // output three digits
}
else if( v >= 10 )
{
  // output two digits
}
else
{
  // output one digit
}
```

Reply:
Oh...

Reply:
Did this snippets come before or after my "Oh"? I think the way I did it is almost the same but I think it's less redundant because the last bit (`// output one digit`) always needs to happen anyway. Unless I'm missing some cases.